Sources/MessagingPush/PushHandling/PushEventHandlerProxy.swift

```diff
@@ -37,7 +37,7 @@ class PushEventHandlerProxyImpl: PushEventHandlerProxy {
     }
 
     func addPushEventHandler(_ newHandler: PushEventHandler) {
-        nestedDelegates[String(describing: newHandler)] = newHandler
+        nestedDelegates[newHandler.identifier] = newHandler
```

Comment:
The bug fix. 

At runtime, multiple `UNUserNotificationCenterDelegateWrapper` instances will call this function. Using `identifier` makes sure that each handler is uniquely identified.